doc/content/verification_and_validation/verification_standards.md

```diff
@@ -0,0 +1,36 @@
+# Ensuring Clear and Consistent Verification in TMAP8
+
+To guarantee that all verification cases in TMAP8 are readily grasped, transparent in their execution, and maintain a consistent format, we've established these standardization guidelines. It's important to note that all verification cases also serve as MOOSE tests, although sometimes with simplified meshes and larger timesteps. These requirements apply to four critical aspects of the verification/test system:
+
+## Input files
+
+1. +Comprehensive Comments+: Include clear comments within the input file explaining its purpose and any specific requirements for running it successfully.
+2. +Constants declaration+: Define all constant values used in the input file at the start of the input file, with self documenting variable names wherever possible.
+3. +Documented Units+: Ensure all values within the input file have their units clearly documented using the units functionality in [input file syntax](https://mooseframework.inl.gov/application_usage/input_syntax.html). Use the same functionality for unit conversions. Additional details on the units system can be found in the [MOOSE units](https://mooseframework.inl.gov/source/utils/Units.html) documentation. As a rule, use the easiest-to-read units and convert them to the required units for the numerical solve.
+4. +Outputs+: By default, all outputs required for verification should be enabled. These can be selectively disabled within the test specification for each specific test as needed.
+5. +Streamlined Input Files+: Whenever possible, leverage [command-line arguments (CLI)](https://mooseframework.inl.gov/moose/application_usage/command_line_usage.html) to manage input files with minor variations instead of creating duplicate files.
```

Comment:
```suggestion
5. +Streamlined Input Files+: Whenever possible, leverage [command-line arguments (CLI)](https://mooseframework.inl.gov/moose/application_usage/command_line_usage.html) to manage input files with minor variations instead of creating duplicate files. For an example, see [/ver-1a/tests].
```